README.md

```diff
@@ -13,25 +13,57 @@
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/justeat/fozzie.svg)](https://lgtm.com/projects/g/justeat/fozzie/alerts/)
 
+## What is Fozzie?
+
+Fozzie is an SCSS Helper Library that's used to help ensure web projects across JET have access to a set of baseline SCSS variables, mixins and functions.
+
+By including this helper library, the consuming web application will have access to our shared PIE Design tokens, as well as common SCSS helper mixins and functions for things like font-size, spacing and setting media queries.
+
 ## Usage
 
-1. The easiest way to use fozzie modules in your Sass setup is to use [Eyeglass](https://www.npmjs.com/package/eyeglass).
+### Pre-requisites
+
+To use the fozzie SCSS helper library, you'll need to ensure a couple of things:
+
+1. That you use `dart-sass` to compile your Sass. The [sass](https://www.npmjs.com/package/sass) module uses dart-sass by default now, so if you use the latest version of this module, you'll be good-to-go.
+
+   `node-sass` support in Sass has been officially deprecated and as this library uses up-to-date Sass syntax (namely `@use` and `@forward`, rather than `@import`), it won't work when compiling with `node-sass`.
 
-If you are using the [fozzie gulp build tasks](https://www.npmjs.com/package/@justeat/gulp-build-fozzie), then Eyeglass is automatically setup ready to use.  If not, you can use it in one of the following ways:
+2. Your build tool supports importing via the `node_modules` folder.
 
-- [Gulp](https://github.com/sass-eyeglass/eyeglass/blob/master/site-src/docs/integrations/gulp.md)
-- [WebPack](https://github.com/sass-eyeglass/eyeglass/issues/153#issuecomment-300895607)
+   Both Webpack and Parcel support this by setting `includePaths` to point to the `node_modules` folder. More info on setting this up in your project can be found in the FAQ's (TODO: Add Link to docs).
 
-2.  Install the fozzie module using NPM or Yarn:
+### Installation
+
+1.  Install the fozzie module using NPM or Yarn:
 
     ```bash
     yarn add @justeat/fozzie
     ```
 
-3.  Then within your Sass files, you will need to import this module.
+2.  Then within your Sass files, you will need to import this module.
 
     ```scss
-    @import 'fozzie';
+    @use 'fozzie' as f;
     ```
 
-You can then use the base styles that `fozzie` has available in this module (and that will be documented shortly).
+Once you have imported fozzie into your Sass, you'll have access to the fozzie variables, mixins and functions, which can be used as in the following example:
+
+  ```scss
+    .myCoolComponent {
+      // Using PIE Variables
+      background: f.$color-background-default;
+      border-radius: f.$radius-rounded-b;
+
+      // Using helper mixins
+      @include font-size('body-l');
+
+      // Using helper functions
+      padding: spacing('b');
+
+      // Using media query helper
+      @include media('>mid') {
+        padding: spacing('c');
```

Comment:
Wondering if these should all be like this?
```suggestion
      @include f.font-size('body-l');

      // Using helper functions
      padding: f.spacing('b');

      // Using media query helper
      @include media('>mid') {
        padding: f.spacing('c');
```

Reply:
@xander-marjoram Good spot – was just testing reviewers with that one 😉

Reply:
fixed.